src/ln/channel.rs

```diff
@@ -2040,6 +2046,14 @@ impl Channel {
 
 	}
 
+	/// Set channel fields with remote peer forwarding parameters, used mainly by ChannelDetails to feed the Router
+	pub fn channel_update(&mut self, msg: &msgs::ChannelUpdate) {
+		self.their_to_self_delay = msg.contents.cltv_expiry_delta;
```

Comment:
to_self_delay is the time before their commitment transaction loses the CSV delay, CLTV_EXPIRY_DELTA is the required difference between an incoming HTLC and the corresponding outgoing HTLC, they're very different values.

Reply:
urgh, ugly from my part, corrected, was thinking to_self_delay was CLTV_EXPIRY_DELTA while writing this by distraction, even if I'm pretty sure I used it correctly as CSV in others times